styles/global.less

```diff
@@ -40,6 +40,10 @@
     font-size: large;
 }
 
+.x-large {
```

Comment:
Consider moving this before the `.large` so they are ordered biggest to smallest in the file.

Reply:
Whoops, I should have noticed that to begin with. Done